website/website/index.html

```diff
@@ -103,11 +103,6 @@ <h3>1. Pick an area on the map.</h3><p>Click on a marker, when it turns red, it
 				<h3>2. Choose data quality report.</h3>
 				<div class="select">
 					<select class="minimal" id="cardtype"  required>
-						<option value="Report" selected>Report ...</option>
```

Comment:
I think this is the default value to be displayed in the selection form. This should probalby stay or should it not?